test/test_dependencies.jl

```diff
@@ -0,0 +1,47 @@
+using Mimi
+
+#list of URLs of branches of packages to test
+dependencies = ["https://github.com/davidanthoff/fund.jl/archive/master.zip"]
+
+#list of failed tests to build as you go
+errors = []
+
+#make a temporary directory to run the tests in
+run(`mkdir tmp_testing`)
+cd("tmp_testing")
+
+#loop through each dependent package
+for url in dependencies
+  #download the package
+  run(`wget $url`)
+  zip_name = chomp(readall(`basename $url`))
+  run(`unzip -x $zip_name`)
+  run(`rm $zip_name`)
+  package_name = chomp(readall(`ls`))
+  #test the package
+  try
+    Pkg.test(package_name)
```

Comment:
This won't work because the git repos like fund.jl are not actually packages. Here you'll actually have to shell out to julia itself and pass the name of the script that should be run to julia.